WooCommerce/WooCommerce.xcodeproj/project.pbxproj

```diff
@@ -1878,8 +1876,6 @@
 			dstPath = "";
 			dstSubfolderSpec = 10;
 			files = (
-				26D9E54528C107F80098DF26 /* WooFoundation.framework in Embed Frameworks */,
-				26D9E54928C10A3C0098DF26 /* Experiments.framework in Embed Frameworks */,
```

Comment:
This is the change / fix: Do not embed these frameworks in the widgets extension, because the main app already embeds them.

I'm relatively sure this will solve our upload problems because I already run into this in the past. I also verified the change using the Archive > Validate feature in Xcode.

**Before**

<img width="1136" alt="Screen Shot 2022-09-05 at 10 27 43 am" src="https://user-images.githubusercontent.com/1218433/188340942-e5f41b88-45b3-4a3b-8105-3d3d134d2575.png">

**After**

<img width="1136" alt="Screen Shot 2022-09-05 at 10 31 53 am" src="https://user-images.githubusercontent.com/1218433/188340953-34be3f88-f573-447e-a807-625c839791c5.png">


Unfortunately, one can never be to sure with Apple and ASC, so we'll know only when the [release build](https://buildkite.com/automattic/woocommerce-ios/builds/7674#01830b20-18fd-4cef-bedd-5d494da79152) finishes.

Reply:
Update: The ASC upload worked

<img width="1186" alt="image" src="https://user-images.githubusercontent.com/1218433/188342335-98340a75-4054-4520-bf3d-45966134bdc5.png">

Reply:
If the issue was really that a target other than the app target had another target marked as "Embed and Sign", then I'm surprised that my protection against that from https://github.com/woocommerce/woocommerce-ios/pull/4226 (internal ref: paNNhX-ee-p2#how-to-avoid-this-in-the-future) didn't warn and protect about it.
Maybe that check / build phase needs to be updated to include this case that was missed?

Reply:
At first, I thought maybe this had something to do with the build phase orders, as I noticed that Xcode appended the "Embed App Extensions" phase to the list, resulting in the check running before it.

![image](https://user-images.githubusercontent.com/1218433/188426877-67e80002-2035-4ca5-b2c8-087e34208643.png)

I reordered them, but the result was the same: No error.

I thought maybe it was a Debug vs Release thing, then. But even when Archiving from Xcode I didn't get an error.

I haven't verified this, but I wonder if the path the script looks into, or the query it uses, might result in app extensions being ignored.

Reply:
Hmmm that could be. I think it'll be worth debugging this later when you find some time, by looking at the query and trying to run it step by step after intentionally restoring the incorrect state prior to this PR. Even if it's not a blocker for this new widget target anymore, it could save our bacon in the future.